### PR TITLE
Adding initial phone number as we do initialEmail

### LINKED ISSUE #69: Prefill Phone Number
Is it possible to prefill the phone number? I know Account Kit itself allows this, but I'm not seeing anything for it in this libraries configuration.

Thanks!

### PR DESCRIPTION
**Pull request by anilreddykatta**
Closes #69 

1. Added `initialPhone` in configurations
2. Updated `ios` & `android` files with similar change
3. Updated docs

### CODE REVIEW

**Comment by gaguirre** [comment]
Hi @anilreddykatta, thanks for this PR! :)
Did you test it in both platforms?

**Comment by anilreddykatta** [comment]
@gaguirre I tested in `ios` looks good. I don't have access to android phones, in simulator looks good!

File: ios/RNAccountKitViewController.m [hunk 1]
```diff
@@ -49,11 +49,11 @@ - (void)loginWithPhone: (RCTPromiseResolveBlock)resolve
 {
     _resolve = resolve;
     _reject = reject;
+    NSString *prefillPhone - self.initialPhone;
```
Comment (gaguirre) [inline ios/RNAccountKitViewController.m]: Is this ok? Seems like should be a `=` instead of a `-`

Comment (gaguirre) [inline ios/RNAccountKitViewController.m]: @anilreddykatta could you see this?

Comment (anilreddykatta) [inline ios/RNAccountKitViewController.m]: @gaguirre Yep! I fixed this locally thought I pushed this change! I will work on this over the weekend and do a test on android too! Thanks for pointing this out!

**Comment by gaguirre** [comment]
Sorry for the delay @anilreddykatta 
I'll test this before merge it and publish a new version to npm.

**Comment by anilreddykatta** [comment]
@gaguirre No problem! Thanks for the update!

**Comment by gaguirre** [comment]
I'm not able to run the example in iOS and Android neither.
Please, update the RN0.40 example or add a new one with a working version.
I'll not merge this until I can test it 😕

**Comment by nadimtuhin** [comment]
Hey guys I also need this :p

**Comment by anilreddykatta** [comment]
@gaguirre Sure. I am on it! I will update asap.

**Comment by anilreddykatta** [comment]
@gaguirre Thanks for feedback. It seems now account kit doesn't accept `NSString` for phone number. I added `AFKPhoneNumber` initialization logic. Let me know what you think about it? I tested this in `IOS`. Please, can you provide me some insights into `android`, I work mostly on `IOS`.

**Comment by agungjk** [comment]
Is it already released?

**Comment by gaguirre** [comment]
@anilreddykatta I've tested and it's failing for both Android and iOS.
However, I'll create a new PR based on your branch with the fixes. Thanks for your help.

**Pull request closed without merging**